exe.py

```diff
@@ -0,0 +1,128 @@
+from Conta import Conta
+from Extrato import Extrato
+from Cliente import Cliente
+
+resp = ""
+users = []
+accs = []
+id_user = -1
+
+
+while resp != "nao":
+
+    erro = True
+    found = False
+
+    resp = input("Deseja realizar qual operação? ")
+
+    if (resp == "cliente"):
+
+        nome = input("Qual seu nome? ")
+
+        while erro == True:
+            try:
+                cpf = int(input("Qual seu cpf (somente numeros)? "))
+                erro = False
+
+            except:
+                print("Opa! Digite somente números!")
+                erro = True
+
+        end = input("Qual seu endereço? ")
+        user = Cliente(nome, cpf, end)
+        users.append(user)
+
+        print("Confirme suas informações:")
+        print(f'\nNome: {nome}\nCpf: {cpf}\nEndereço: {end}\n')
+        correto = input("Correto(s/n)? ")
+        if correto == "s":
+            id_user += 1
+            print(f'Ok! Perfil criado com sucesso com a ID:{id_user}')
+            verif = input("Gostaria de verificar as informações? ")
+            if (verif == "s"):
+                print(f'\nNome: {nome}\nCpf: {cpf}\nEndereço: {end}\n')
+
+            elif (verif == "n"):
+                continue
+
+            else:
+                print("Não entendemos sua resposta. Continuando...")
+                continue
+
+        elif correto == "n":
+            continue
+
+        else:
+            print("Não entendemos sua resposta. Continuando...")
+            continue
+
+    elif (resp == "mostrar"):
+
+        for u in range(len(users)):
+            print(f'Nome: {users[u].nome} | CPF: {users[u].cpf} | Endereço: {users[u].endereco}')
+
```

Comment:
Pra ficar mais fácil de entender o código, seria legal mover cada "bloco" pra uma função

```suggestion
        mostrar(users)
```

Reply:
É a boa! Tava fazendo isso. Ja fiz o de criar cliente, funciona show.